app/javascript/application.js

```diff
@@ -1,6 +1,7 @@
 import './src/jquery'
 
 import { initAll } from 'govuk-frontend'
+import { initAll as initAllMoJ } from '@ministryofjustice/frontend'
```

Comment:
Does this impact asset performance?
As we are only using sub nav and sortable, are we able to load only the resources we need?

Reply:
I dont know if we can. I can't find a way to obviously do it in the application.js file

I've measured the largest contentful paint load in chrome and seems to be very similar. I've also tested with network throttling and compared it to main uat and it seems about the same.